tables_app/src/main/java/org/opendatakit/tables/fragments/TableManagerFragment.java

```diff
@@ -211,53 +240,53 @@ public boolean onContextItemSelected(MenuItem item) {
     String localizedDisplayName = selectedStruct.getLocalizedDisplayName();
 
     switch (item.getItemId()) {
-    case R.id.table_manager_delete_table:
-      AlertDialog confirmDeleteAlert;
-      // Prompt an alert box
-      AlertDialog.Builder alert = new AlertDialog.Builder(this.getActivity());
-      alert.setTitle(getString(R.string.confirm_remove_table))
-          .setMessage(getString(R.string.are_you_sure_remove_table, localizedDisplayName));
-      // OK Action => delete the table
-      alert.setPositiveButton(getString(R.string.yes), new DialogInterface.OnClickListener() {
-        public void onClick(DialogInterface dialog, int whichButton) {
-          // treat delete as a local removal -- not a server side deletion
-          DbHandle db = null;
-          try {
+      case R.id.table_manager_delete_table:
```

Comment:
Please undo formatting changes like this. They make it harder to see what code has actually changed, and harder to review history.